.github/workflows/pages.yaml

```
@@ -46,7 +59,12 @@ jobs:
       - name: Build
         # https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
         # GHA runners have 2 CPUs
-        run: make site -j2
+        if: env.REVISIONS == 'unset'
+        run: make site -j2 PROTOTYPE_BRANCHES_PREFIX="${{ env.PROTOTYPE_BRANCHES_PREFIX }}"
```

B, maybe nitpicky: To keep things a bit more readable, could we try folding flags into one:
```suggestion
        run: |
          make site -j2 \
            ${{ github.event.inputs.revisions != '' && format('REVISIONS={0}', github.event.inputs.revisions) }} \
            ${{ github.event.inputs.revisions == '' && github.event.inputs.prefix && format('PROTOTYPE_BRANCHES_PREFIX={0}', github.event.inputs.prefix) }}
```

This has the major benefit of making the default value still controlled by the Makefile. If adopted the `env` block could be removed.

---

I am happy with that change.

---

Adjusted per feedback in https://github.com/usnistgov/OSCAL-Reference/pull/23/commits/2a7ced9018dd1a6bc0a40415aa4792ae55a2a682.